go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/pilosa/pilosa
 
+replace github.com/hashicorp/memberlist => github.com/pilosa/memberlist v0.1.4-0.20190408132233-ff8741fd3108
```

Comment:
i was not aware of "replace", that's actually a really nice feature. anyway, i don't know enough to be sure, but this looks reasonable and we can pretty easily see if we keep hitting that specific failure.

Reply:
see also https://github.com/rogpeppe/gohack